src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final int numberOfColor = new Random().nextInt(Color.values().length);
```

Comment:
the problem is that when you create `new ColorSupplier()` - this variable is initialized and is no further changed
so your number is not randomly generated each time you call the `getRandomColor()` method on the same ColorSupplier object - and this is a mistake

please move this variable to the method - so it could be reset each time you call the method